src/Raven.Server/ServerWide/Commands/ModifyConflictSolverCommand.cs

```diff
@@ -1,30 +1,48 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 using Raven.Client.Documents;
-using Raven.Server.Json;
+using Raven.Server.Rachis;
 using Sparrow.Json;
 using Sparrow.Json.Parsing;
 
 namespace Raven.Server.ServerWide.Commands
 {
     public class ModifyConflictSolverCommand : UpdateDatabaseCommand
     {
-        public BlittableJsonReaderObject Value;
+        public string DatabaseResolverId;
+        public BlittableJsonReaderObject ResolveByCollection;
+        public bool ResolveToLatest;
 
         public ModifyConflictSolverCommand():base(null){}
 
         public ModifyConflictSolverCommand(string databaseName) : base(databaseName){}
         
         public override void UpdateDatabaseRecord(DatabaseRecord record, long etag)
         {
-            record.ConflictSolverConfig = JsonDeserializationServer.ConflictSolver(Value);
+            record.ConflictSolverConfig.DatabaseResolverId = DatabaseResolverId;
+            record.ConflictSolverConfig.ResolveToLatest = ResolveToLatest;
+
+            record.ConflictSolverConfig.ResolveByCollection = new Dictionary<string, ScriptResolver>();
```

Comment:
I think it would be clearer in the code if we just strongly typed objects.
this is rarely called, we can pay the extra serialization / deserialization costs